packages/webcert/src/feature/certificate/Buttons/ReplaceCertificateButton.tsx

```diff
@@ -37,25 +39,26 @@ const ReplaceCertificateButton: React.FC<Props> = ({ name, description, enabled,
       <>
         <InfoBox type="observe" activateIconWrap>
           <p>
-            {!isDodsbevis
-              ? 'Om intyget innehåller ett allvarligt fel, till exempel om det är utfärdat på fel patient, bör du istället makulera intyget.'
-              : 'Om dödsbeviset är utfärdat på fel patient ska du istället makulera dödsbeviset.'}
+            {isDodsbevis || isDodsorsaksIntyg
+              ? `Om ${certificate}et är utfärdat på fel patient ska du istället makulera ${certificate}et.`
+              : 'Om intyget innehåller ett allvarligt fel, till exempel om det är utfärdat på fel patient, bör du istället makulera intyget.'}
           </p>
         </InfoBox>
         <p>
           Ett intyg kan ersättas om det innehåller felaktiga uppgifter eller om ny information tillkommit efter att intyget utfärdades. När
           ett intyg ersätts med ett nytt skapas ett utkast, med samma information som i det ursprungliga intyget, som du kan redigera innan
           du signerar intyget.
         </p>
-        {isDodsbevis && (
-          <>
-            <p>
-              Senast skapade dödsbevis är det som gäller. Om du ersätter det tidigare dödsbeviset och lämnar in det nya så blir det därför
-              detta dödsbevis som gäller.
-            </p>
-            <p>Det nya utkastet skapas på den enhet du är inloggad på.</p>
-          </>
-        )}
+        {isDodsbevis ||
+          (isDodsorsaksIntyg && (
```

Comment:
This could be changed to `(isDodsbevis || isDodsorsaksIntyg) && ( ...` for a bit more readability.